doc/modules/classes.rst

```diff
@@ -161,28 +160,34 @@ Classes
    :toctree: generated/
    :template: class.rst
 
-   cross_validation.KFold
-   cross_validation.LabelKFold
-   cross_validation.LabelShuffleSplit
-   cross_validation.LeaveOneLabelOut
-   cross_validation.LeaveOneOut
-   cross_validation.LeavePLabelOut
-   cross_validation.LeavePOut
-   cross_validation.PredefinedSplit
-   cross_validation.ShuffleSplit
-   cross_validation.StratifiedKFold
-   cross_validation.StratifiedShuffleSplit
-
+   model_selection.KFold
```

Comment:
I think the splitters should be demarcated from the rest. There's precedent for splitting these functionally in `datasets`.